PURL-TYPES.rst

```diff
@@ -301,6 +315,32 @@ maven
       pkg:maven/net.sf.jacob-projec/jacob@1.14.3?classifier=x64&type=dll
 
 
+mlflow
+------
+``mlflow`` for MLflow ML models (Azure ML, Databricks, etc.)
+
+- The repository is the MLflow tracking URI. There is no default. Examples:
+
+  - Azure ML: ``https://<region>.api.azureml.ms/mlflow/v1.0/subscriptions/<subscription-id>/resourceGroups/<resource-group-name>/providers/Microsoft.MachineLearningServices/workspaces/<workspace-name>``
+  - Azure Databricks: ``https://adb-<numbers>.<number>.azuredatabricks.net/api/2.0/mlflow``
+  - AWS Databricks: ``https://dbc-<alphanumeric>-<alphanumeric>.cloud.databricks.com/api/2.0/mlflow``
+  - GCP Databricks: ``https://<numbers>.<number>.gcp.databricks.com/api/2.0/mlflow``
+  
+- The ``namespace`` is empty.
+- The ``name`` is the model name. Case sensitivity depends on the server implementation:
+
+  - Azure ML: it is case sensitive and must be kept as-is in the package URL.
+  - Databricks: it is case insensitive and must be lowercased in the package URL.
+
+- The ``version`` is the model version.
+- Known qualifiers keys are: ``model_uuid`` and ``run_id`` as defined in the MLflow documentation.
+- Examples::
```

Comment:
All these examples should have a `repository_url` since there is no default.

Reply:
Updated MLflow spec + test to always have `repository_url`